### DISCUSSION TITLE
not sycning for few days but can connect to peers.

### DISCUSSION THREAD

**Discussion by hhenrry501** [Chia-Network/chia-blockchain#5575] [Support]
I can see peer nodes in GUI, but there's no download/upload speed. And sync status is always syncing but the peak hight hasnt been changing....

I got this in debug.log:
2021-05-20T12:00:44.234 full_node chia.rpc.rpc_server     : WARNING  Sending data failed. Exception Traceback (most recent call last):
  File "chia\rpc\rpc_server.py", line 67, in _state_changed
  File "aiohttp\client_ws.py", line 150, in send_str
  File "aiohttp\http_websocket.py", line 687, in send
  File "aiohttp\http_websocket.py", line 643, in _send_frame
  File "aiohttp\http_websocket.py", line 660, in _write
ConnectionResetError: Cannot write to closing transport

Is there anything I could do to solve this?

**Comment by BrandtH22** [comment]
@hhenrry501 , the newest version(s) of chia has fixes in place to resolve this issue. Please download it from here: https://www.chia.net/downloads/

Since this issue has been resolved with the latest version(s) of chia we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
